mlir/test/Dialect/Tosa/tosa-infer-shapes.mlir

```diff
@@ -1309,15 +1309,3 @@ func.func @test_large_constant_permutation() {
   %72 = tosa.transpose %14, %cst_26 : (tensor<?x27xi64>, tensor<2xi32>) -> tensor<?x27xi64>
   return
 }
-
-// -----
-
-// CHECK-LABEL: test_rank0_transpose_perms
```

Comment:
This check is now covered by `test_transpose_rank0_perms` in `invalid.mlir` so we can remove it, right?

Reply:
Yes, I think the verifier should now cover all cases where a rank-0 tensor could be supplied as permutation tensor.